packages/ui-brand/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@hackoregon/ui-brand",
+  "sideEffects": false,
+  "version": "0.0.4",
+  "description": "Brand assets for CIVIC",
+  "main": "dist/index.js",
+  "modules": "es/index.js",
+  "scripts": {
+    "build:esm": "rimraf es && babel src --out-dir es --copy-files --no-comments",
+    "build:cjs": "rimraf dist && babel src --out-dir dist --copy-files --no-comments",
```

Comment:
It'd be nice to get to the point where we don't need to different build targets like this.

Reply:
Will be addressed in #1208